src/test/shared/featureController.test.ts

```diff
@@ -12,27 +12,45 @@ describe('FeatureController', async () => {
     const testConfig = new TestSettingsConfiguration()
     const activeSettingKey = 'activeSetting'
     const inactiveSettingKey = 'inactiveSetting'
+    const activeNotBoolSettingKey = 'activeNotBoolSetting'
+    const inactiveNotBoolSettingKey = 'inactiveNotBoolSetting'
     const missingSettingKey = 'missingSetting'
     const variableSettingKey = 'variableSetting'
     const permanentSettingKey = 'permanentSetting'
     const permanentMissingSettingKey = 'permanentMissingSetting'
     const permanentMissingVariableSettingKey = 'permanentMissingVariableSetting'
-    await testConfig.writeSetting(`toggle.${activeSettingKey}`, true)
-    await testConfig.writeSetting(`toggle.${inactiveSettingKey}`, false)
-    await testConfig.writeSetting(`toggle.${variableSettingKey}`, false)
-    await testConfig.writeSetting(`toggle.${permanentSettingKey}`, true)
+    let features: FeatureController
 
-    const features = new FeatureController(testConfig, [
-        permanentSettingKey,
-        permanentMissingSettingKey,
-        permanentMissingVariableSettingKey
-    ])
+    beforeEach(async () => {
+        await testConfig.writeSetting(`toggle.${activeSettingKey}`, true)
+        await testConfig.writeSetting(`toggle.${inactiveSettingKey}`, false)
+        await testConfig.writeSetting(`toggle.${activeNotBoolSettingKey}`, 'can you hear me')
+        await testConfig.writeSetting(`toggle.${inactiveNotBoolSettingKey}`, 0)
+        await testConfig.writeSetting(`toggle.${variableSettingKey}`, false)
+        await testConfig.writeSetting(`toggle.${permanentSettingKey}`, true)
+        features = new FeatureController(testConfig, [
+            permanentSettingKey,
+            permanentMissingSettingKey,
+            permanentMissingVariableSettingKey
+        ])
+    })
 
-    it('returns whether or not a feature is active', () => {
+    it('returns true if feature is active', () => {
```

Comment:
All of the tests are associated with `isFeatureActive`. You should have a describe ('isFeatureActive') nested within describe ('FeatureController') and all of your test and setup code would go in that scope